lib/darksky.rb

```diff
@@ -0,0 +1,33 @@
+require 'httparty'
+
+class Darksky
+  def self.darksky_info(coordinates)
```

Comment:
If you thought about an instance of `Darksky` or of `GoogleCoordinates` as a session to an external API, what benefits might there be to creating instances?